symengine/sets.cpp

```diff
@@ -90,6 +92,19 @@ RCP<const Set> Interval::close() const
     return interval(start_, end_, false, false);
 }
 
+bool Interval::contains(const RCP<const Basic> &a) const
```

Comment:
Yes.
Pushed it in the recent commit. [Here](https://github.com/nishnik/symengine/blob/e85fc4418df01c0e76c0b804b286886718ad9a2e/symengine/tests/basic/test_sets.cpp#L72)!